### PR TITLE
Add explanation of the keyword 'today'

### PR DESCRIPTION
**Pull request by DanRJ**
https://github.com/Altinn/altinn-studio/issues/7228

### CODE REVIEW

File: schemas/json/layout/layout.schema.v1.json [hunk 1]
```diff
@@ -160,13 +160,13 @@
         "minDate": {
           "type": "string",
           "title": "Minimum allowed date",
-          "description": "Sets the minimum allowed date. Defaults to 1900-01-01T12:00:00.000Z'.",
+          "description": "Sets the minimum allowed date. Can also use keywords such as 'today'. The 'today' keyword will disable all past dates dynamically. Defaults to 1900-01-01T12:00:00.000Z'.",
```
Comment (lorang92) [inline schemas/json/layout/layout.schema.v1.json:163]: `Can also use keywords such as 'today'. The 'today' keyword will disable all past dates dynamically.`
This makes is sound like we support other keywords. I suggest we only reference the once we actually support, and update this if the need be in the future.

Better with something like this?

```suggestion
          "description": "Sets the minimum allowed date. Can also use keyword 'today' to disable all past dates dynamically based on the current date. Defaults to 1900-01-01T12:00:00.000Z'.",
```

**Review by lorang92 — CHANGES_REQUESTED** [review]

File: schemas/json/layout/layout.schema.v1.json [hunk 2]
```diff
@@ -160,13 +160,13 @@
...
           "title": "Minimum allowed date",
-          "description": "Sets the minimum allowed date. Defaults to 1900-01-01T12:00:00.000Z'.",
+          "description": "Sets the minimum allowed date. Can also use keyword 'today' to disable all past dates dynamically based on the current date. Defaults to 1900-01-01T12:00:00.000Z'.",
           "default": "1900-01-01T12:00:00.000Z"
         },
         "maxDate": {
           "type": "string",
           "title": "Maximum allowed date",
-          "description": "Sets the maxiumum allowed date. Defaults to 2100-01-01T12:00:00.000Z.'",
+          "description": "Sets the maxiumum allowed date. Can also use keywords such as 'today'. The 'today' keyword will disable all future dates dynamically. Defaults to 2100-01-01T12:00:00.000Z.'",
```
Comment (lorang92) [inline schemas/json/layout/layout.schema.v1.json:169]: Same fix as suggested her.

**Review by lorang92 — CHANGES_REQUESTED** [review]

File: schemas/json/layout/layout.schema.v1.json (hunk 2 above)
Comment (DanRJ) [reply schemas/json/layout/layout.schema.v1.json]: Fixed

**Review by lorang92 — APPROVED** [review]

**Review by SandGrainOne — APPROVED** [review]

**Review by SandGrainOne — COMMENTED** [review]
Is this ' character necessary?
Both descriptions has an extra ' character at the end. Doesn't look right.

**Comment by DanRJ** [comment]
> Is this ' character necessary? Both descriptions has an extra ' character at the end. Doesn't look right.

I didnt see that one until you pointed it out, I think its missing one in the beginning of the date time example, e.g: '1900-...', however there is a couple of different ways used to display values in this file already, \"value\", or 'value' or no special characters at all, value.

How would you prefer it? @SandGrainOne @lorang92

**Comment by SandGrainOne** [comment]
> How would you prefer it? @SandGrainOne @lorang92
I don't think the quotes adds anything here. At least not in VS code. My suggestion is to remove the unnecessary quote character.

**Pull request merged**